sql/hive/src/main/scala/org/apache/spark/sql/hive/SQLBuilder.scala

```diff
@@ -78,6 +78,27 @@ class SQLBuilder(logicalPlan: LogicalPlan, sqlContext: SQLContext) extends Loggi
     }
   }
 
+  private def toSQL(node: LogicalPlan, topNode: Boolean): String = {
```

Comment:
@gatorsmile You mentioned that you found some regressions. Did you mean some test cases in the current master branch are failing, or you constructed new test cases that failed this PR?